packages/salesforcedx-vscode-soql/package.json

```diff
@@ -32,6 +32,7 @@
     "@salesforce/ts-sinon": "1.2.2",
     "@types/chai": "^4.0.0",
     "@types/debounce": "^1.2.0",
+    "@types/mkdirp": "0.5.2",
```

Comment:
there is version 1.0.1 out now, but npm list mkdirp shows several hits on the 0.5.x versions.  This seem to be coming from the system-tests package.

Reply:
Here's the only reference i see to it.

https://github.com/forcedotcom/salesforcedx-vscode/blob/987031482aeaa0da4a53d0e1e285d9278dd0ce39/packages/system-tests/package.json#L28